bundles/org.openhab.ui/web/src/assets/definitions/widgets/system/slider.js

```diff
@@ -6,7 +6,7 @@ export default () => [
   pd('max', 'Max', 'Maximum value'),
   pd('step', 'Step', 'Minimum interval between values'),
   pd('vertical', 'Vertical', 'Display the slider vertically'),
-  pb('label', 'Display Label', 'Display a label above the slider knob'),
+  pb('label', 'Display Label', 'Display a label above the slider knob while sliding'),
```

Comment:
If you change definitions would you mind running `node generate.js` from https://github.com/openhab/openhab-webui/tree/main/bundles/org.openhab.ui/doc/components/src - this should and will be part of a CI process but... as of now it's not...

Reply:
Thank you for pointing that out. I just did that and it resulted in much more files being changed 😳. Mostly docs on material icons have been added, but e.g. for `oh-input.md` the syntax has even changed (from `:` to `()`). My fault?

Reply:
Most if not all of these changes come as a result of #917 - I forgot to add the same thing back! Thanks!